src/reanimated2/utils.ts

```diff
@@ -14,6 +14,9 @@ export function getRelativeCoords(
 ): ComponentCoords {
   'worklet';
   const parentCoords = measure(parentRef);
+  if (!parentCoords) {
+    return { x, y };
```

Comment:
Can we add a comment here why it is valid to return `{ x, y }` here?

Reply:
Hah, it's not. That was a bug. Really, I think this function should just return `null` if it can't perform the measurement - I added a jsdoc comment and made some other tweaks.